manifests/client/config.pp

```diff
@@ -1,99 +1,92 @@
-# == Class: nfs::client::config
-#
+# Class: nfs::client::config
+# @summary
 # This Function exists to
 #  1. configure nfs as a client
 #
-#
-# === Links
-#
-# * {Puppet Docs: Using Parameterized Classes}[http://j.mp/nVpyWY]
-#
-#
-# === Authors
-#
+# @author
 # * Daniel Klockenkaemper <mailto:dk@marketing-factory.de>
+# * Martin Alfke <tuxmea@betadots.de>
 #
-
 class nfs::client::config {
-
-  if $::nfs::client::nfs_v4 {
-
-    if $::nfs::defaults_file != undef {
-      if $::nfs::client_gssd_options != '' {
-        $_gssd1_aug = ["set ${::nfs::client_gssdopt_name} \"'${::nfs::client_gssd_options}'\""]
+  if $nfs::client::nfs_v4 {
```

Comment:
I'm also a bit torn with all those linting changes, they are really not related to implementing Puppet 8 support, but rather pleasing puppet-lint. Can you maybe provide one PR with the result of `bundle exec rake lint_fix`?